multimodal/src/autogluon/multimodal/utils.py

```diff
@@ -1976,3 +1977,37 @@ def update_config_by_rules(
                 UserWarning,
             )
     return config
+
+
+def process_save_path(path, resume: Optional[bool] = False, raise_if_exist: Optional[bool] = True):
+    """
+    Convert the provided path to an absolute path and check whether it is valid.
+    If a path exists, either raise error or return None.
+    A None path can be identified by the `setup_outputdir` to generate a random path.
+
+    Parameters
+    ----------
+    path
+        A provided path.
+    resume
+        Whether this is a path to resume training.
+    raise_if_exist
+        Whether to raise error if the path exists.
+
+    Returns
+    -------
+    A complete and verified path or None.
+    """
+    path = os.path.abspath(os.path.expanduser(path))
+    if resume:
+        last_ckpt_path = os.path.join(path, LAST_CHECKPOINT)
+        assert os.path.isfile(last_ckpt_path), f"The last checkpoint {last_ckpt_path} doesn't exist."
```

Comment:
Consider to add more details in the Assertion message:

> Trying to resume training from '{path}'. However, it does not contain the last checkpoint file: '{last_ckpt_path}'. Are you using the correct path?

Reply:
Thanks. Updated.